index.html

```diff
@@ -84,76 +84,87 @@ <h2>
-            [=window controls=] by calling the minimize/maximize/close etc APIs
-            directly without the need to use browser's native window controls.
+            <dfn>Additional Window Controls</dfn>: feature to replace [=window
+            controls=] by calling the minimize/maximize/close etc APIs directly
+            without the need to use browser's native window controls.
           </li>
           <li>
             <dfn data-lt="IWA">Isolated Web App</dfn>: feature for installed
             and signed web applications which are packaged as Web Bundles and
-            can be used offline instead of being fetched over HTTPS.
-            See <a href="https://github.com/WICG/isolated-web-apps">explainer</a>.
+            can be used offline instead of being fetched over HTTPS. See
+            <a href="https://github.com/WICG/isolated-web-apps">explainer</a>.
           </li>
         </ul>
       </section>
-
       <section>
         <h3>
           Display mode extensions
         </h3>
         <p>
-          Additionally to the normal
-          <a data-cite="appmanifest#dfn-display-modes-list">display modes</a>,
+          Additionally to the normal <a data-cite=
+          "appmanifest#dfn-display-modes-list">display modes</a>,
           [=manifest/display_override=] also supports certain extensions to it.
         </p>
-        
-        <dt>
-          <dfn data-export="" data-dfn-for="display mode">
-            borderless
-          </dfn>
-        </dt>
-        <dd>
-          The web application does not have any host-native title bar or
-          [=window controls=] visible and with the web contents extended to the
-          whole title bar area. The app can specify [=draggable region=]s in the
-          web contents to create a customized title bar. The user agent may
-          change the title bar state depending on various security
-          considerations, like an out-of-scope navigation.
-        </dd>
-
-        <dt>
-          [=display mode/window-controls-overlay=]
-        </dt>
-
+        <dl>
+          <dt>
+            <dfn data-export="" data-dfn-for="display mode">borderless</dfn>
+          </dt>
+          <dd>
+            The web application does not have any host-native title bar or
+            [=window controls=] visible and with the web contents extended to
+            the whole title bar area. The app can specify [=draggable region=]s
+            in the web contents to create a customized title bar. The user
+            agent may change the title bar state depending on various security
+            considerations, like an out-of-scope navigation.
+          </dd>
+          <dt>
+            [=display mode/window-controls-overlay=]
+          </dt>
+          <dt>
+            <dfn data-export="" data-dfn-for="display mode">tabbed</dfn>
+          </dt>
+          <dd>
+            The web application can have multiple application contexts combined
```

Comment:
Should 'application context' here be a link? Same for the other one in this paragragh and more at line 251 and 252.

Reply:
Done. I won't repeatedly link multiple times per paragraph.